versioned_docs/version-8.5/self-managed/setup/upgrade.md

```diff
@@ -111,6 +111,10 @@ helm search repo camunda/camunda-platform --versions
 
 ### Helm Chart 10.0.0
 
+:::caution Breaking changes
```

Comment:
@christinaausley since I see you are making changes, I have another suggestion. Let's make this a `:::danger` and adjust the title (or remove it).

Reply:
```suggestion
:::danger 
```